<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ApartmentManager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/iconApartment.ico</normaloff>:/icons/icons/iconApartment.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>761</width>
      <height>411</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>125</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>127</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Этаж</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кол-во комнат</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Площадь (м²)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Наличие отделки</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Наличие балкона</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Стоимость (₽)</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNewDatabase"/>
    <addaction name="actionLoadDatabase"/>
    <addaction name="actionSaveDatabase"/>
    <addaction name="actionSaveAsDatabase"/>
    <addaction name="actionMergeDatabase"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuWrite">
    <property name="title">
     <string>Запись</string>
    </property>
    <addaction name="actionAddAp"/>
    <addaction name="actionEditAp"/>
    <addaction name="actionDelAp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWrite"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewDatabase"/>
   <addaction name="actionLoadDatabase"/>
   <addaction name="actionSaveDatabase"/>
   <addaction name="actionSaveAsDatabase"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNewDatabase">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/file_negate.png</normaloff>:/icons/icons/file_negate.png</iconset>
   </property>
   <property name="text">
    <string>Новый файл</string>
   </property>
   <property name="toolTip">
    <string>Создание базы данных</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLoadDatabase">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/open_negate.png</normaloff>:/icons/icons/open_negate.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Загрузка базы данных из файла</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveDatabase">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save_negate.png</normaloff>:/icons/icons/save_negate.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранение базы данных в файл</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/exit_negate.png</normaloff>:/icons/icons/exit_negate.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из приложения</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help_negate.png</normaloff>:/icons/icons/help_negate.png</iconset>
   </property>
   <property name="text">
    <string>Помощь</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/about_negate.png</normaloff>:/icons/icons/about_negate.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddAp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/add_negate.png</normaloff>:/icons/icons/add_negate.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEditAp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/edit_negate.png</normaloff>:/icons/icons/edit_negate.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionDelAp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/delete_negate.png</normaloff>:/icons/icons/delete_negate.png</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMergeDatabase">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/merge_negate.png</normaloff>:/icons/icons/merge_negate.png</iconset>
   </property>
   <property name="text">
    <string>Объединение БД</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSaveAsDatabase">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/save_as_negate.png</normaloff>:/icons/icons/save_as_negate.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
